src/data/sym/basic.lean

```diff
@@ -201,7 +201,7 @@ begin
     congr, },
 end
 
-instance subsingleton (n : ℕ) [subsingleton α] : subsingleton (sym α n) :=
+instance (n : ℕ) [subsingleton α] : subsingleton (sym α n) :=
```

Comment:
This argument order is more idiomatic.
```suggestion
instance [subsingleton α] (n : ℕ) : subsingleton (sym α n) :=
```
I *think* but I'm not sure, that it also matters in some TC search heuristics.